py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -36,6 +35,7 @@ def __init__(
         serialized_engine: Optional[bytes] = None,
         input_binding_names: Optional[List[str]] = None,
         output_binding_names: Optional[List[str]] = None,
+        output_shapes: Optional[List[Tuple[int]]] = None,
```

Comment:
Why is this needed? shouldnt the engine have the shapes?

Reply:
during compile stage, we have the output shapes information, however this information is not saved to the TorchTensorRTModule, when user call torch_tensorrt.save() we run_shape_analysis() again to get the output shapes, which is not only unnecessary, also run_shape_analysis requires the Engine which won't work for the cross runtime compilation feature.
Hence @dheerajperi  suggested that I should skip run_shape_analysis in the torch_tensorrt.save()
instead save the output shaped in compile stage.